lisp/l/coordinates.l

```diff
@@ -222,7 +222,7 @@
 	 ((or (memq wrt '(:parent parent :world world nil))
 	      (eq wrt *world-coords*)) v)
 	 ((coordinates-p wrt)   (transform  (send wrt :worldrot) v))
-	 (t (send self :error ":parent-orientation wrt?" wrt))))
+	 (t (send self :error ":parent-orientation wrt? wrt must be either :world, :local or :parent" wrt))))
```

Comment:
coordinate instance is also supported as wrt.

Reply:
Thank you for your advice. I pushed additional commit.